src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -269,25 +238,45 @@ namespace AirflowNetworkBalanceManager {
         {
         }
 
-        void calc(int const ZoneNum,
-                  int const SurfNum,
-                  int const PrevOpeningstatus,
-                  Real64 const TimeOpenDuration,
-                  Real64 const TimeCloseDuration,
+        void calc(int ZoneNum,
+                  Real64 TimeOpenDuration,
+                  Real64 TimeCloseDuration,
                   int &OpeningStatus,
                   int &OpeningProbStatus,
                   int &ClosingProbStatus); // function to perform calculations
 
-        bool openingProbability(int const ZoneNum,
-                                Real64 const TimeCloseDuration); // function to perform calculations of opening probability
+        bool openingProbability(int ZoneNum,
+                                Real64 TimeCloseDuration); // function to perform calculations of opening probability
 
-        bool closingProbability(Real64 const TimeCloseDuration); // function to perform calculations of closing probability
+        bool closingProbability(Real64 TimeCloseDuration); // function to perform calculations of closing probability
     };
 
-    extern Array1D<OccupantVentilationControlProp> OccupantVentilationControl;
-
 } // namespace AirflowNetworkBalanceManager
 
+    struct AirflowNetworkBalanceManagerData : BaseGlobalStruct {
```

Comment:
@mitchute When I split the implementation up into the current subdirectory form, my intent was to circle back and handle some of this stuff at a later date. I'd propose that instead of naming this struct `AirflowNetworkBalanceManagerData` that we name it `AirflowNetworkBalanceManager` instead and get rid of the `AirflowNetworkBalanceManager` namespace. That may leave some of the objects in a bit of limbo right now, but we can clean that up later.

Reply:
@jasondegraw If all of these functions were member methods of an object class, then I that could work. But that would leave all of this hanging out on the ```EnergyPlus``` namespace. I'm not sure if this is going to be problematic. I'll defer to others on this one.

Reply:
I think we should make one dedicated pass on the airflow network side of things, and do it next, right away.  I'd like for this chunky PR to get dropped in as-is if possible.

Reply:
@Myoldmopar OK, I'll start up a housekeeping branch with some miscellaneous fixes, leaving the manager alone. Once this is merged, I'll pull back in develop and work on the manager a bit. It shouldn't take too long to get into shape to go in.